src/cli/README_NETDATA.md

```diff
@@ -157,9 +159,36 @@ Print netdata help menu.
 ```bash
 > netdata help
 help
+publish
 register
 show
 steeringdata
+unpublish
+Done
+```
+
+### publish
+
+The Network Data Publisher provides mechanisms to limit the number of similar Service and/or Prefix (on-mesh prefix or external route) entries in the Thread Network Data by monitoring the Network Data and managing if or when to add or remove entries.
+
+This command requires `OPENTHREAD_CONFIG_NETDATA_PUBLISHER_ENABLE`.
+
+The following formats are used to publish DNS/SRP Service entry (available when `OPENTHREAD_CONFIG_TMF_NETDATA_SERVICE_ENABLE` is enabled):
+
+- `netdata publish dnssrp anycast <serial-num>` to publish "DNS/SRP Service Anycast Address" with a given serial number to be published in the Thread Network Data.
+- `netdata publish dnssrp unicast <address> <port>` to publish "DNS/SRP Service Unicast Address" with given address and port number info. The address/port info is included in Service TLV data.
+- `netdata publish dnssrp unicast <port>` to publish "DNS/SRP Service Unicast Address" with given port number and the device's mesh-local EID for the address. The address and port info is included in Server TLV data.
+
+A `netdata publish dnssrp` command will remove and replace any previous "DNS/SRP Service" entry that was being published (from earlier `netdata publish dnssrp` commands).
```

Comment:
Is this command implemented?

Reply:
I think you mean the reference to `netdata publish dnssrp` in the above statement. 
I mean any flavor of this command (with any args). Will update the doc to make it clear.